market_maker_keeper/uniswapv2_market_maker_keeper.py

```diff
@@ -121,10 +121,10 @@ def __init__(self, args: list, **kwargs):
         self.arguments = parser.parse_args(args)
         setup_logging(self.arguments)
 
-        if self.arguments.rpc_host.startswith("http"):
-            endpoint_uri = f"{self.arguments.rpc_host}:{self.arguments.rpc_port}"
-        else:
+        if self.arguments.rpc_host.startswith("https"):
             endpoint_uri = f"https://{self.arguments.rpc_host}:{self.arguments.rpc_port}"
```

Comment:
That still won't work with node URIs where the port cannot be suffixed to the end.  Recommend using pymaker's [ web3_via_http](https://github.com/makerdao/pymaker/blob/master/pymaker/__init__.py#L53) method.